### PR TITLE
BDM173 - BMI DAI staking

### PR DESCRIPTION
**Pull request by Arvolear**
All tests and linters pass.

### CODE REVIEW

File: contracts/PolicyBook.sol [hunk 1]
```diff
@@ -33,16 +40,28 @@ contract PolicyBook is IPolicyBook, ERC20 {
...
+    
+    daiToken = IERC20(_contractsRegistry.getContract(_contractsRegistry.getDAIName()));
+    bmiDaiStakingAddress = contractsRegistry.getContract(contractsRegistry.getBmiDAIStakingName());    
+  }  
+
+  function approveAllDaiTokensForStakingAndTransferOwnership() external onlyOwner {
+    bool _success = daiToken.approve(bmiDaiStakingAddress, MAX_INT);
+    require(_success, "Failed to approve DAI tokens");
+
+    transferOwnership(bmiDaiStakingAddress);
```
Comment (Armen-Arakelian) [inline contracts/PolicyBook.sol:64]: why is this needed?

Comment (Arvolear) [reply contracts/PolicyBook.sol:64]: Yep, this one Is needed so functions in ERC1155MintableBurable could work

Comment (Armen-Arakelian) [reply contracts/PolicyBook.sol:64]: that's for tokens, understood
